docs/management/connectors/action-types/servicenow-itom.asciidoc

```diff
@@ -7,6 +7,11 @@
 
 The ServiceNow ITOM connector uses the https://docs.servicenow.com/bundle/rome-it-operations-management/page/product/event-management/task/send-events-via-web-service.html[Event API] to create ServiceNow events.
 
+[float]
+[[servicenow-itom-connector-prerequisites]]
+==== Prerequisites
+After upgrading to 7.16, you must create an integration user in ServiceNow and assign it the `personalize_choices` and `evt_mgmt_integration` roles.
```

Comment:
ITOM is introduced in 7.16. I think is better to remove the `After upgrading to 7.16`.

Reply:
Inserting the following as discussed on Slack.

Create an integration user in SN and assign it the following roles.

- `personalize_choices`: Allows the user to retrieve Choice elements options like Severity
- `evt_mgmt_integration`: Enables integration with external event sources by allowing the user to create events